core/shared/src/main/scala/laika/directive/std/BreadcrumbDirectives.scala

```diff
@@ -56,19 +57,26 @@ object BreadcrumbDirectives {
       @tailrec
       def entriesFor(tree: TreeCursor,items:List[NavigationItem] = Nil) :List[NavigationItem] = {
         val title = tree.target.title.getOrElse(SpanSequence(tree.path.name))
+        
         val item = context.newNavigationItem(title, tree.target.titleDocument, Nil, tree.target.targetFormats)
+        
         tree.parent match {
           case None => item :: items
           case Some(parent) => entriesFor(parent,item ::items)
         }
       }
 
-      val docEntry = {
+      // if cursor is targeting title document, we don't need to append docEntry
+      // as Seq[NavigationItem] generated by `entriesFor` contains link to title document
+      if(cursor.target.path.name.eqv(TitleDocumentConfig.defaultInputName)) {
```

Comment:
Ah, I've noticed I should use title document name from config as user may choose another name but README🤔

Reply:
I think the easiest way to check this is `if (cursor.parent.titleDocument.exists(_.path == cursor.path))`

Reply:
That seems much simpler. Thanks!